src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -74,6 +89,22 @@ public void run() {
                     }
                 }
             }
+            for (Player player : Bukkit.getOnlinePlayers()) {
+                if (!TeamUtils.getTeamByPlayer(player).isObserver()) {
+                    if (time == 3) {
```

Comment:
@dentmaged That would send a message every tick. While it wouldn't show since you're dividing by an integer, meaning you'll get an integer (3, 2, 1), it's needless and could be made more efficient. Perhaps just put it in the `if` for the countdown, which only executes every time the time is divisible by 20 (one second).